tools/docker-network/tests/rewards_test.go

```diff
@@ -208,59 +210,58 @@ func Test_DelayedClaimingRewards(t *testing.T) {
 		time.Sleep(secToWait)
-		d.ClaimRewardsForDelegator(ctx, account, delegationOutputID1)
+		d.ClaimRewardsForDelegator(ctx, delegatorWallet, delegationOutputData)
 	}
 
 	{
 		// delegate funds to V2
-		delegationOutputID, _ := d.DelegateToValidator(account.ID, d.Node("V2").AccountAddress(t))
+		delegationOutputData := d.DelegateToValidator(delegatorWallet, d.Node("V2").AccountAddress(t))
 
 		// delay claiming rewards in the same slot of delegation
-		delegationOutputID1, _ := d.DelayedClaimingTransition(ctx, account, delegationOutputID)
-		d.AwaitCommitment(delegationOutputID1.CreationSlot())
+		delegationOutputData = d.DelayedClaimingTransition(ctx, delegatorWallet, delegationOutputData)
+		d.AwaitCommitment(delegationOutputData.ID.CreationSlot())
 
 		// the delegated stake should be 0, thus poolStake should be equal to validatorStake
 		v2Resp, err := clt.Validator(ctx, d.Node("V2").AccountAddress(t))
 		require.NoError(t, err)
 		require.Equal(t, v2Resp.PoolStake, v2Resp.ValidatorStake)
 
 		// wait until next epoch to destroy the delegation
-		d.ClaimRewardsForDelegator(ctx, account, delegationOutputID1)
+		d.ClaimRewardsForDelegator(ctx, delegatorWallet, delegationOutputData)
 	}
 }
 
-func issueCandidacyPayloadInBackground(d *DockerTestFramework, accountID iotago.AccountID, startSlot, endSlot iotago.SlotIndex, slotDuration uint8) {
+func issueCandidacyPayloadInBackground(d *DockerTestFramework, wallet *mock.Wallet, startSlot, endSlot iotago.SlotIndex, slotDuration uint8) {
 	go func() {
-		fmt.Println("Issuing candidacy payloads for account", accountID, "in the background...")
-		defer fmt.Println("Issuing candidacy payloads for account", accountID, "in the background......done")
+		fmt.Println("Issuing candidacy payloads for account", wallet.BlockIssuer.AccountData.ID, "in the background...")
+		defer fmt.Println("Issuing candidacy payloads for account", wallet.BlockIssuer.AccountData.ID, "in the background......done")
 
 		for i := startSlot; i < endSlot; i++ {
-			d.IssueCandidacyPayloadFromAccount(accountID)
+			d.IssueCandidacyPayloadFromAccount(wallet)
 			time.Sleep(time.Duration(slotDuration) * time.Second)
 		}
 	}()
 }
 
-func issueValidationBlockInBackground(wg *sync.WaitGroup, d *DockerTestFramework, accountID iotago.AccountID, startSlot, endSlot iotago.SlotIndex, blocksPerSlot int) {
+func issueValidationBlockInBackground(wg *sync.WaitGroup, wallet *mock.Wallet, startSlot, endSlot iotago.SlotIndex, blocksPerSlot int) {
 	wg.Add(1)
 
 	go func() {
 		defer wg.Done()
-		fmt.Println("Issuing validation block for account", accountID, "in the background...")
-		defer fmt.Println("Issuing validation block for account", accountID, "in the background......done")
-		clt := d.wallet.DefaultClient()
+		fmt.Println("Issuing validation block for account", wallet.Name, "in the background...")
+		defer fmt.Println("Issuing validation block for account", wallet.Name, "in the background......done")
```

Comment:
This should perhaps be modified to say wallet rather than account or print the actual account id.